shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -435,7 +464,25 @@ - (nullable id)accessibilityElementAtIndex:(NSInteger)index {
   if (index == 0) {
     return _semanticsObject;
   }
+
+  // Create an addtional child to act as the accessibility container for the platform view.
+  if (_semanticsObject.node.IsPlatformViewNode() && index == [self accessibilityElementCount] - 1) {
+    shell::FlutterPlatformViewsController* controller = _bridge.get()->GetPlatformViewsController();
+    if (controller) {
+      FlutterPlatformViewSemanticsContainer* child = [[FlutterPlatformViewSemanticsContainer alloc]
```

Comment:
Instead of recreating this every time you're asked for it I wonder if it should be cached? iOS queries these things fairly frequently.

Also: Is this creating a memory leak? Who is cleaning these up again?

Reply:
@goderbauer Fixed.